src/ts/components/dashboard/Transaction.tsx

```diff
@@ -0,0 +1,3 @@
+/*
```

Comment:
remove this?

Reply:
yup

Reply:
I will have a `TransactionHistory` component to be placed in the dashboard. So yes this can be removed.

Reply:
ok